spark-image-classification/.gitignore

```diff
@@ -0,0 +1 @@
+out/
```

Comment:
Can push one level down, we are going to more spark demo projects.
/spark/image-classification/